paf-mvp-core-js/src/express/node.ts

```diff
@@ -211,6 +209,10 @@ export class Node implements INode {
           break;
       }
 
+      const correlationId = req.header('onekey-correlation-id');
+      const sender = req.header('referer') || req.header('origin');
+      this.logError(endPointName, typedError, correlationId, sender);
```

Comment:
I am not sure it deserves a dedicated method:
- it's called only once
- but more importantly, I believe this log **should not be called** outside of `catchErrors`: only this method should be allowed to log errors. In other words, if the code is inline here, there is no risk another method reuses it.